charts/kafka-topic-replicator/container/Dockerfile

```
@@ -0,0 +1,15 @@
+FROM python:3.10.0-slim-bullseye as base
+
+ENV POETRY_VERSION=1.1.12
+
+RUN pip install "poetry==$POETRY_VERSION"
+COPY app/* /app/
+COPY poetry.lock pyproject.toml ./
+RUN poetry install
```

```suggestion
COPY poetry.lock pyproject.toml ./
RUN poetry install
COPY app/* /app/
```
I think it makes sense to reorder these, that way you do not run poetry whenever the source code changes.